src/Akeneo/Platform/Bundle/ImportExportBundle/front/src/components/StorageConfigurator/SftpStorageConfigurator.tsx

```diff
@@ -69,9 +67,16 @@ const SftpStorageConfigurator = ({
   const portValidationErrors = filterErrors(validationErrors, '[port]');
   const publicKey = useGetPublicKey();
   const passwordIsStoredOnServer = storage.login_type === 'password' && storage.password === undefined;
+  const notify = useNotify();
 
   const canCopyToClipboard = (): boolean => 'clipboard' in navigator;
-  const copyToClipboard = (publicKey: string) => canCopyToClipboard() && navigator.clipboard.writeText(publicKey);
+  const copyToClipboard = (publicKey: string) => {
+    canCopyToClipboard() && navigator.clipboard.writeText(publicKey);
```

Comment:
We need to early return here because the notification will be send even the user cannot copy to clipboard

```suggestion
    if (!canCopyToClipboard()) {
        return;
    }
    
    navigator.clipboard.writeText(publicKey);
```